scripts/Interface/ui/drill/model/specifications.py

```diff
@@ -0,0 +1,212 @@
+            FIGARO: [REFL_NPOL]
+            }
+
+    # parameters for each acquisition mode
+    COLUMNS = {
+            SANS_ACQ: [
+                "SampleRuns",
+                "SampleTransmissionRuns",
+                "AbsorberRuns",
+                "BeamRuns",
+                "FluxRuns",
+                "ContainerRuns",
+                "ContainerTransmissionRuns",
+                "TransmissionAbsorberRuns",
+                "TransmissionBeamRuns",
+                "MaskFiles",
+                "ReferenceFiles",
+                "OutputWorkspace",
+                "WedgeWorkspace",
+                "SampleThickness",
+                "CustomOptions"
+                ],
+            OMEGA_SCAN: [
+                ],
+            REFL_POL: [
+                "Run",
+                "Run00",
+                "Run01",
+                "Run10",
+                "Run11",
+                "DirectRun",
+                "OutputWorkspace",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions"
+                ],
+            REFL_NPOL: [
+                "Run",
+                "DirectRun",
+                "OutputWorkspace",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions"
+                ]
+            }
+
+    # algo name for each acquisition mode
+    ALGORITHM = {
+            SANS_ACQ:   "SANSILLAutoProcess",
+            OMEGA_SCAN: None,
+            REFL_POL:   "ReflectometryILLAutoProcess",
+            REFL_NPOL:  "ReflectometryILLAutoProcess"
+            }
+
+    # settings for each acquisition mode
+    SETTINGS = {
+            SANS_ACQ : {
+                "ThetaDependent": True,
+                "SensitivityMaps": "",
+                "DefaultMaskFile": "",
+                "NormaliseBy": "Timer",
+                "SampleThickness": "0.1",
+                "BeamRadius": "0.05",
+                "WaterCrossSection": "1",
+                "OutputType": "I(Q)",
+                "CalculateResolution": "None",
+                "DefaultQBinning": "PixelSizeBased",
+                "BinningFactor": "1",
+                "OutputBinning": "",
+                "NPixelDivision": "1",
+                "NumberOfWedges": "0",
+                "WedgeAngle": "30",
+                "WedgeOffset": "0",
+                "AsymmetricWedges": False,
+                "MaxQxy": "0",
+                "DeltaQ": "0",
+                "IQxQyLogBinning": False
+                },
+            OMEGA_SCAN : {
+                },
+            REFL_POL : {
+                "PolarizationEfficiencyFile": "",
+                "DirectFlatBackground": "Background Constant Fit",
```

Comment:
In the branch the default option is going to be "Background Average" which is much more faster.

Reply:
I changed it.